src/main/java/io/github/pylonmc/pylon/base/PylonBlocks.java

```diff
@@ -61,6 +64,17 @@ private PylonBlocks() {
     );
     static {
         GRINDSTONE.register();
+        Grindstone.Recipe.RECIPE_TYPE.addRecipe(new Grindstone.Recipe(
+            pylonKey("string_from_bamboo"),
+            new RecipeChoice.ExactChoice(new ItemStack(Material.BAMBOO)),
+            3,
```

Comment:
```suggestion
            4,
```
OPTIONAL: 3 is an awkward number in minecraft, I prefer to just work in powers of 2 where possible / makes sense, but this is ultimately my preference so it's your choice whether you take this